UPSTREAM_README.md

```diff
@@ -94,15 +86,24 @@ If you’d like to start your own Python project from scratch, you can either co
 
 - Adjust the Dependabot settings in `.github/dependabot.yaml` to your desired target branch that you’d like to have monitored by Dependabot.
 
-To develop your new package, create a [virtual environment](https://docs.python.org/3/tutorial/venv.html) and install its `dev`,  `test` and `docs` dependencies:
+To develop your new package, create a [virtual environment](https://docs.python.org/3/tutorial/venv.html) and install its `dev`,  `test` and `docs` dependencies. By default, the [Makefile](https://www.gnu.org/software/make/manual/make.html#toc-An-Introduction-to-Makefiles) sets up a [Python 3.10](https://www.python.org/downloads/release/python-3100/) virtual environment in `.venv/` and initializes the local [git hooks](#git-hooks):
+```bash
+make setup
+```
+
+If you’d like to use another Python version, then the following works:
+
+```bash
+PYTHON=python3.9 make setup
+```
+
+And finally, activate your virtual environment (the period here is a [shorthand for the `source` command](https://www.gnu.org/savannah-checkouts/gnu/bash/manual/html_node/Bourne-Shell-Builtins.html#index-_002e)):
+
 ```bash
-python3.10 -m venv .
-source ./bin/activate
-pip install --upgrade pip
-pip install --editable .[dev,test,docs]
+. .venv/bin/activate
```

Comment:
Doesn't `make setup` already activate the `venv`?

Reply:
Every recipe in a Makefile runs in its own subprocess, and so when the Makefile is done the process is done and with it the activated environment for that process.

This line activates the venv for the shell process itself.